src/prefect/engine.py

```diff
@@ -1192,10 +1189,13 @@ async def suspend_flow_run(
 
 @sync_compatible
 @inject_client
+@experimental_parameter(
```

Comment:
While I was in here I noticed that I'd incorrectly applied the experimental decorator to the wrong method, should be here on `suspend` rather than the private method above.